src/components/Note/NoteReactionList/NoteReactionBar/NoteReactionBar.test.tsx

```diff
@@ -1,36 +1,45 @@
-import {render, fireEvent, screen} from "@testing-library/react";
+import {fireEvent, render, screen} from "@testing-library/react";
 import userEvent from "@testing-library/user-event";
 import {REACTION_EMOJI_MAP} from "types/reaction";
+import {Provider} from "react-redux";
+import {ApplicationState} from "types";
+import getTestApplicationState from "utils/test/getTestApplicationState";
+import getTestStore from "utils/test/getTestStore";
 import {NoteReactionBar} from "./NoteReactionBar";
 
+const renderNoteReactionBar = (close?: () => void, click?: () => void, overwrite?: Partial<ApplicationState>) =>
+  render(
+    <Provider store={getTestStore(overwrite)}>
+      <NoteReactionBar closeReactionBar={close ?? jest.fn()} reactions={[]} handleClickReaction={click ?? jest.fn()} />
+    </Provider>
+  );
+
 describe("NoteReactionBar", () => {
   it("renders the reaction buttons correctly", () => {
-    render(<NoteReactionBar closeReactionBar={() => {}} reactions={[]} handleClickReaction={() => {}} />);
-
+    renderNoteReactionBar();
     REACTION_EMOJI_MAP.forEach((emoji) => {
-      expect(screen.getByText(emoji)).toBeInTheDocument();
+      expect(screen.getByText(emoji.emoji)).toBeInTheDocument();
     });
   });
 
   it("calls handleClickBar when a reaction button is clicked", () => {
-    const handleClickReaction = jest.fn();
-    render(<NoteReactionBar closeReactionBar={() => {}} reactions={[]} handleClickReaction={handleClickReaction} />);
+    const handleClickReactionFunction = jest.fn();
+    renderNoteReactionBar(undefined, handleClickReactionFunction);
     fireEvent.click(screen.getByText("👍"));
-    expect(handleClickReaction).toHaveBeenCalled();
+    expect(handleClickReactionFunction).toHaveBeenCalled();
   });
```

Comment:
could you add a test where you overwrite the `skinTone` state and see if the 👍 colored correctly?